gradle/common/java11Options.gradle

```diff
@@ -17,7 +17,6 @@
  */
 
 project.ext.JDK11_OPTIONS = ['-XX:-MaxFDLimit',
-                             '-Djdk.attach.allowAttachSelf=true',
```

Comment:
this I believe is unnecessary, but I understand this is unrelated to the PR , so I can revert if there are reservations

Reply:
I think bytebuddy and mockito need the option. Let's not change it in this patch.

Reply:
yeah agreed